packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -0,0 +1,549 @@
+        transform: 'translate(-50%, 0)',
+        left: '50%',
+        top: 'initial',
+        width: 2,
+      },
+      '&$jumped': {
+        backgroundColor: colors.focused,
+      },
+    },
+    trackBefore: {
+      zIndex: 1,
+      left: 0,
+      backgroundColor: colors.primary,
+      transition: commonTransitions,
+      '&$focused, &$activated, &$jumped': {
+        backgroundColor: colors.primary,
+      },
+    },
+    trackAfter: {
+      right: 0,
+      backgroundColor: colors.secondary,
+      transition: commonTransitions,
+      '&$vertical': {
+        bottom: 0,
+      },
+    },
+    /* Thumb styles  */
+    thumb: {
+      position: 'absolute',
+      zIndex: 2,
+      transform: 'translate(-50%, -50%)',
+      width: 12,
+      height: 12,
+      borderRadius: '50%',
+      transition: commonTransitions,
+      backgroundColor: colors.primary,
+      '&$focused': {
+        boxShadow: `0px 0px 0px 9px ${fade(colors.primary, 0.16)}`,
+      },
+      '&$activated': {
+        width: 17,
+        height: 17,
+        transition: 'none',
+      },
+      '&$disabled': {
+        cursor: 'no-drop',
+        width: 9,
+        height: 9,
+        backgroundColor: colors.disabled,
+      },
+      '&$zero': {
+        border: `2px solid ${colors.disabled}`,
+        backgroundColor: 'transparent',
+      },
+      '&$focused$zero': {
+        border: `2px solid ${colors.focused}`,
+        backgroundColor: fade(colors.focused, 0.34),
+        boxShadow: `0px 0px 0px 9px ${fade(colors.focused, 0.34)}`,
+      },
+      '&$activated$zero': {
+        border: `2px solid ${colors.focused}`,
+      },
+      '&$jumped': {
+        width: 17,
+        height: 17,
+      },
+    },
+    focused: {},
+    activated: {},
+    disabled: {},
+    zero: {},
+    vertical: {},
+    reverse: {},
+    jumped: {},
+  };
+};
+
+const KEY_CODES = {
+  PAGE_UP: 33,
+  PAGE_DOWN: 34,
+  END: 35,
+  HOME: 36,
+  ARROW_LEFT: 37,
+  ARROW_UP: 38,
+  ARROW_RIGHT: 39,
+  ARROW_DOWN: 40,
+};
+
+function addEventListener(node, event, handler, capture) {
+  node.addEventListener(event, handler, capture);
+  return {
+    remove: function remove() {
+      node.removeEventListener(event, handler, capture);
+    },
+  };
+}
+
+function percentToValue(percent, min, max) {
+  return (max - min) * percent / 100 + min;
+}
+
+function roundToStep(number, step) {
+  return Math.round(number / step) * step;
+}
+
+function getOffset(node) {
+  const { scrollY, scrollX } = global;
+  const { left, top } = node.getBoundingClientRect();
+
+  return {
+    top: top + scrollY,
+    left: left + scrollX,
+  };
+}
+
+function getMousePosition(event) {
+  if (event.changedTouches && event.changedTouches[0]) {
+    return {
+      x: event.changedTouches[0].pageX,
+      y: event.changedTouches[0].pageY,
+    };
+  }
+
+  return {
+    x: event.pageX,
+    y: event.pageY,
+  };
+}
+
+const calculatePercent = (node, event, isVertical, isReverted) => {
```

Comment:
You are using a `function` with the other helpers, not an arrow, maybe we should be consistent.

Reply:
Fixed it